test/unit/expiration-year.js

```diff
@@ -14,93 +14,110 @@ function yearsFromNow(fromNow, digits) {
+        [yearsFromNow(5, 2), {isValid: true, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(10, 2), {isValid: true, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(11, 2), {isValid: true, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(12, 2), {isValid: true, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(19, 2), {isValid: true, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(20, 2), {isValid: false, isPotentiallyValid: false, isCurrentYear: false}],
+        [yearsFromNow(25, 2), {isValid: false, isPotentiallyValid: false, isCurrentYear: false}],
+        [yearsFromNow(33, 2), {isValid: false, isPotentiallyValid: false, isCurrentYear: false}]
+      ],
+
+      // This doesn't take 20xx -> 21xx into account, but probably YAGNI
+      'accepts three-digit years': [
+        [yearsFromNow(-3).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(-1).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(0).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(1).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(5).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(11).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(17).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}],
+        [yearsFromNow(23).slice(0, 3), {isValid: false, isPotentiallyValid: true, isCurrentYear: false}]
+      ]
+    };
+
+    Object.keys(describes).forEach(function (key) {
+      var tests = describes[key];
+
+      describe(key, function () {
+        tests.forEach(function (test) {
+          var arg = test[0];
+          var output = test[1];
+
+          it('returns ' + JSON.stringify(output) + ' for "' + arg + '"', function () {
+            expect(expirationYear(arg)).to.deep.equal(output);
+          });
         });
       });
     });
   });
+
+  describe('maxElapsedYear', function () {
+    it('defaults maxElapsedYear is 19', function () {
+      expect(expirationYear(yearsFromNow(19))).to.deep.equal({isValid: true, isPotentiallyValid: true, isCurrentYear: false});
+      expect(expirationYear(yearsFromNow(20))).to.deep.equal({isValid: false, isPotentiallyValid: false, isCurrentYear: false});
+    });
+
+    it('accepts maxElapsedYear', function () {
+      expect(expirationYear(yearsFromNow(20), 20)).to.deep.equal({isValid: true, isPotentiallyValid: true, isCurrentYear: false});
+    });
+
+    it('returns invalid if beyond maxElapsedYear', function () {
+      expect(expirationYear(yearsFromNow(21), 20)).to.deep.equal({isValid: false, isPotentiallyValid: false, isCurrentYear: false});
+    });
+  });
```

Comment:
`describe('maxElapsedYear'` is the main test in `expiration-year.js`.
The other difference is only indents 🙏 
https://github.com/braintree/card-validator/pull/54/files?w=1 will make a more readable difference.